admin/src/main/scala/io/buoyant/admin/names/DelegateHandler.scala

```diff
@@ -47,7 +47,7 @@ class DelegateHandler(
         <script id="dtab-base-data" type="application/json">${DelegateApiHandler.Codec.writeStr(dtabBase)}</script>
       """,
       javaScripts = Seq("dtab_viewer.js", "delegator.js", "delegate.js"),
-      csses = Seq("styleguide/styleguide.css", "styleguide/local.css", "admin.css", "delegator.css", "dashboard-shared.css")
+      csses = Seq("admin.css", "delegator.css", "dashboard-shared.css", "dashboard.css")
```

Comment:
if dashboard-shared is going to be shared across all pages, should we add it to AdminHandler instead?